test/celeritas/em/CoulombScattering.test.cc

```diff
@@ -331,25 +310,38 @@ TEST_F(CoulombScatteringTest, simple_scattering)
         angle.push_back(dot_product(this->direction(), result.direction));
     }
 
+    static double const expected_angle[] = {1,
+                                            0.99999999776622,
+                                            0.99999999990987,
+                                            0.99999999931707,
+                                            1,
+                                            0.9999999952274,
+                                            0.99999999905465,
+                                            0.99999999375773,
+                                            1,
+                                            0.99999999916491};
+    static double const expected_energy[] = {200,
+                                             199.99999999847,
+                                             199.99999999994,
+                                             199.99999999953,
+                                             200,
+                                             199.99999999673,
+                                             199.99999999935,
+                                             199.99999999572,
+                                             200,
+                                             199.99999999943};
```

Comment:
Maybe we should be testing this at a lower energy (or a range thereof?) With this small a change in direction and energy would we be able to tell if there were a bug?

Reply:
Good point, a range of energies would be better.